### PR TITLE
Update documentation for supply and demand constraints in biofuel model

### PR DESCRIPTION
**Pull request by AlbertLee125**
This pull request updates the documentation for the supply and demand constraints in the biofuel model. It includes setting up the parameters of the distance and the coordinates of the supermarket and site, as well as adding documentation on the parameter and references. Additionally, it adds documentation of the disjunct and the disjunctions.

### CODE REVIEW

File: gdplib/biofuel/model.py [hunk 1]
```diff
@@ -5,43 +5,119 @@
```
Comment (bernalde) [inline gdplib/biofuel/model.py:18]: from __future__ import division is not needed (it was required for Python 2)

File: gdplib/biofuel/model.py [hunk 2]
```diff
@@ -5,43 +5,119 @@
 
 import pandas as pd
```
Comment (bernalde) [inline gdplib/biofuel/model.py:19]: Missing file header

File: gdplib/biofuel/model.py [hunk 3]
```diff
@@ -5,43 +5,119 @@
...
+    summation,
+    TransformationFactory,
+    value,
+    Var,
+)
 from pyomo.gdp import Disjunct
 
 
 def build_model():
+    """_summary_
```
Comment (bernalde) [inline gdplib/biofuel/model.py]: __sumary__?

**Review by bernalde — CHANGES_REQUESTED** [review]

File: gdplib/biofuel/model.py [hunk 4]
```diff
@@ -5,43 +5,119 @@
...
+    Pyomo.ConcreteModel
+        The Pyomo concrete model which descibes the multiperiod location-allocation optimization model designed to determine the most cost-effective network layout and production allocation to meet market demands.
+
+    References
+    ----------
+    [1] Lara, C. L., Trespalacios, F., & Grossmann, I. E. (2018). Global optimization algorithm for capacitated multi-facility continuous location-allocation problems. Journal of Global Optimization, 71(4), 871-889. https://doi.org/10.1007/s10898-018-0621-6
+    [2] Chen, Q., & Grossmann, I. E. (2019). Effective generalized disjunctive programming models for modular process synthesis. Industrial & Engineering Chemistry Research, 58(15), 5873-5886. https://doi.org/10.1021/acs.iecr.8b04600
+    """
     m = ConcreteModel()
     m.bigM = Suffix(direction=Suffix.LOCAL)
```
Comment (ZedongPeng) [inline gdplib/biofuel/model.py]: It might be better if we set the default `BigM =7000` here.

File: gdplib/biofuel/model.py (hunk 1 above)
Comment (AlbertLee125) [reply gdplib/biofuel/model.py:18]: I removed the from future import division. Since the line is outdated.

File: gdplib/biofuel/model.py (hunk 3 above)
Comment (AlbertLee125) [reply gdplib/biofuel/model.py]: Added the description.

File: gdplib/biofuel/model.py (hunk 4 above)
Comment (AlbertLee125) [inline gdplib/biofuel/model.py]: I added the name on the ConCreteModel.

File: gdplib/biofuel/model.py (hunk 2 above)
Comment (AlbertLee125) [reply gdplib/biofuel/model.py:19]: Added the file header.

File: gdplib/biofuel/model.py (hunk 4 above)
Comment (AlbertLee125) [reply gdplib/biofuel/model.py]: I use initialize to set default value as 7000.

**Review by ZedongPeng — APPROVED** [review]

**Review by bernalde — APPROVED** [review]

**Pull request merged**